charts/kueue/values.yaml

```
@@ -227,6 +227,8 @@ kueueViz:
     env:
       - name: KUEUEVIZ_ALLOWED_ORIGINS
         value: "frontend.kueueviz.local"
+      - name: GIN_MODE
+        value: "debug"
```

```suggestion
      - name: GIN_MODE
        value: "release"
```

Shouldn't we set release by default?

---

Probably yes, but we need to be careful if then it wouldn't break something, because iiuc currently we release as "debug" ? :)

---

Or maybe we should modify this variable in helm-chart-package.sh for the release?

---

We could, the question is what is the user-observable difference

---

IIUC, if we specify `GIN_MODE=release`, the CORS origin domain validation does not allow `*`.
So, the default `GIN_MODE=release` might be inconvenient.